2022-06-21 14:50:00,160 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-06-21 14:50:00,162 INFO supervisord started with pid 9
2022-06-21 14:50:01,164 INFO spawned: 'nginx' with pid 12
2022-06-21 14:50:01,166 INFO spawned: 'app' with pid 13
Bottle v0.12.21 server starting up (using WSGIRefServer())...
Listening on http://localhost:5000/
Hit Ctrl-C to quit.

2022-06-21 14:50:02,581 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-06-21 14:50:02,581 INFO success: app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
127.0.0.1 - - [21/Jun/2022 14:50:25] "POST /api/setup HTTP/1.0" 200 0
[CTRL] POST /setup
[Time: 21.06.22 14:50:25] [Level: info] id: fb6d2ac9da8ca5d4
[Time: 21.06.22 14:50:25] [Level: info] coordinator: False
[Time: 21.06.22 14:50:25] [Level: info] clients: ['f9685aa0f6b6d85f', '3ff10e33f147d23b', 'fb6d2ac9da8ca5d4', 'a5d3b3026795de5f', '6987661933051bb3', '2bdaaaf052d5ee8f', 'afa11a1311ef2c80', 'dff74649eef72a35']
[Time: 21.06.22 14:50:25] [Level: info] state: initial
[Time: 21.06.22 14:50:25] [Level: info] transition: read
[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 14:50:25] "GET /api/status HTTP/1.0" 200 126
[Time: 21.06.22 14:50:26] [Level: info] state: read
[Time: 21.06.22 14:50:26] [Level: info] Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 169, in guarded_run
    self.run()
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 184, in run
    transition = self.current_state.run()
  File "/app/states.py", line 62, in run
    self.send_data_to_coordinator()
TypeError: send_data_to_coordinator() missing 1 required positional argument: 'data'

[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 14:50:28] "GET /api/status HTTP/1.0" 200 133
